packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -113,28 +115,33 @@ export function computePerformanceResourceDetails(
   }
 
   // Make sure a connection occurred
-  if (connectEnd !== fetchStart) {
+  if (fetchStart < connectEnd) {
```

Comment:
Note: those conditions were changed (ex: from `fetchStart !== connectEnd` to `fetchStart < connectEnd`) and are not behind the flag, but it should not have any impact, because some other conditions are checked in `toValidEntry`). Ex:

Before this PR, we check that `fetchStart <= connectEnd` in `toValidEntry`, then check that `fetchStart !== connectEnd` here in `computePerformanceResourceDetails`.

With this PR, without the flag, we still check that `fetchStart <= connectEnd` in `toValidEntry`, then check that `fetchStart < connectEnd` here in `computePerformanceResourceDetails`, so this is equivalent.

With this PR, with the flag, we won't have `toValidEntry` anymore, we just check that `fetchStart < connectEnd`, which is still the same.